vue/components/ui/molecules/select/select.vue

```diff
@@ -19,9 +19,27 @@
             </option>
         </select>
         <div class="select-container" v-bind:style="style" v-else>
+            <input-ripe
+                class="select-input"
+                v-bind:value.sync="filterText"
+                v-show="filter && visibleData"

```

Comment:
Move this to a computed, and use `!stuff` in the `v-show` of the select-button below, so it's clear that it's either one or the other.